api/hypertable_size.md

```diff
@@ -51,3 +58,13 @@ Get size information for all hypertables.
 SELECT hypertable_name, hypertable_size(format('%I.%I', hypertable_schema, hypertable_name)::regclass)
   FROM timescaledb_information.hypertables;
 ```
+
+Get size of a continuous aggregate.
```

Comment:
```suggestion
Get the size of a continuous aggregate:
```

Reply:
I think in this case the other sentences should be fixed, too. I specifically copied the weird style of the two examples before 😅

```
Get size information for a hypertable.
Get size information for all hypertables.
Get size of a continuous aggregate.
```

Reply:
the 'information' and 'for' in those earlier sentences mean that the definite article is not required. You can either make them all "Get the size of ..." or "Get size information for ...".